sonar-core/src/main/resources/org/sonar/core/component/db/ComponentMapper.xml

```diff
@@ -156,7 +156,7 @@
     </where>
   </select>
 
-  <select id="selectAllViewsAndSubViews" resultType="hashmap">
+  <select id="selectAllViewsAndSubViews" resultType="UuidWithProjectUuid">
```

Comment:
The method name "selectAllViewsAndSubViews" is clear, it loads views, so it should not accept qualifier parameters. Two options:
- hardcode values of qualifiers in SQL request
- declare a more generic method like selectUuidsForQualifiers(String... qualifiers)